web/rest/client/config/api/raw/provider.yml

```diff
@@ -536,6 +536,41 @@ Ivoz\Provider\Domain\Model\ConferenceRoom\ConferenceRoom:
       required:
       - company
 
+Ivoz\Provider\Domain\Model\Contact\Contact:
+  properties:
+    workPhoneE164:
+      attributes:
+        swagger_context:
+          readOnly: true
+    mobilePhoneE164:
+      attributes:
+        swagger_context:
+          readOnly: true
+    user:
+      attributes:
+        swagger_context:
+          readOnly: true
+  attributes:
+    access_control: >-
+      "ROLE_COMPANY_ADMIN" in roles
+      && user.isVpbxAdmin()
+      && user.hasAccessPrivileges(_api_resource_class, request.getMethod())
+    read_access_control:
+      ROLE_COMPANY_ADMIN:
+        company:
+          eq: "user.getCompany().getId()"
+    write_access_control:
+      - and:
+          - company:
+              eq: "user.getCompany().getId()"
+      - or:
+          - inheritedOrNull:
+              user: 'Ivoz\Provider\Domain\Model\User\User'
+    swagger_context:
+      required:
+        - company
```

Comment:
We don't expose company in client API, do we?

Reply:
This has been copied from other endpoints like Ddi or Extension that also have that at the end.

Reply:
It should not be necessary in any of them, does it change API spec?

Reply:
This has NOT been updated with suggested change to keep same configuration in all endpoints.